resources/static/js/Login.js

```diff
@@ -0,0 +1,31 @@
+// Se definio la variable asignandole el tag por ID del formulario
+var formEmployee = document.getElementById("formPass");
+// Se definieron variables de los campos pass y Id
+var pasword = document.getElementById("EmployeeId");
+var id = document.getElementById("EmployeePass");
+// Regex para campos vacios
+var whiteSpace = /^\s+$/;
+/**
+ * @descriptor Esta funcion va a ser ejecutada cuando se accione el boton submit del formulario
+ * 
+ * @param  {event}
+ */
+formEmployee.onsubmit = function(event){
+	/*
+	   Recorre los elementos del formulario y verifica por cada elemento del mismo
+	   que no este vacio ni con espacios en blanco, previniendo la accion por default del formulario
+	   y mostrando el aviso apropiado
+	*/
+	for (var i = 0; i < formEmployee.length; i++) 
+	{
```

Comment:
La llave del for y del if de adentro están en una nueva línea, mientras que en function(event) está en la misma línea.
Recuerden mantener la coherencia.